### ISSUE TITLE
Spacing within rich text

### ISSUE THREAD

**Issue by inesdgomes** [international-labour-organization/designsystem#1059]
Hi @beatrizmartinmartins One of the questions I think the DS is not totally clear on yet is the spacing within rich text. For example, what should be the spacing before and after each heading level? Wouldn't it be useful to have a page on Figma showing all the spacing for the different components that can be shown inside a rich text?

**Comment by beatrizmartinmartins** [comment]
@inesdgomes Does this make sense? https://www.figma.com/design/qr8jQnxqJkRJFQnVC5PmrB/ILO-Foundations?node-id=1905-4&t=74nmw3gRWJiZHrQM-1 I made a small documentation and included all styles that we have right now inside of the Wrapper so we could see how it behaves within the container. Let me know if this is what we needed.

**Comment by inesdgomes** [comment]
Hi @beatrizmartinmartins Yes, it does, thank you. Wondering if we shouldn't be a bit more generous with spacing between texts (maybe 24px instead of 16px?), especially the text after headings. I find it odd that that spacing is smaller than the spacing between regular body paragraphs, which is 18px.

**Comment by beatrizmartinmartins** [comment]
Would it make sense to have 24px and 64px on desktop and 16px and 56px on Mobile? I think 24px on Mobile is too much @inesdgomes Also changed the guidelines on the Foundations file

**Comment by inesdgomes** [comment]
Yes, I think it works better. A few questions/comments:
- Should the sentence at the top where it says that space between texts should be 16px be changed to 24px then?
- If so, I think the space between paragraphs is still not entirely clear. If you look for example at the paragraph spacing of the Desktop/Body/Regular, that's set at 18px; and Desktop/Body/Small is set at 16px. That's only visible when we try to edit the style - see screenshot below. Should that setting be changed to 24px on all desktop styles and to 16px on all mobile styles?
- We should show spacing for lists, even if the standard spacing applies.

![Image](https://github.com/international-labour-organization/designsystem/assets/22772102/498c587c-cf20-4100-b9ef-98a8bf9add97)

**Comment by beatrizmartinmartins** [comment]
You are right. Just changed everything said above and also changed the typography tokens to only reflect 16 and 24px. @inesdgomes

**Comment by beatrizmartinmartins** [comment]
Fixed all the comments and I'm also showing the lists [here](https://www.figma.com/design/qr8jQnxqJkRJFQnVC5PmrB/ILO-Foundations?node-id=1905-4&t=pX2dJREI5OlmTy4l-1) @inesdgomes

**Comment by inesdgomes** [comment]
@beatrizmartinmartins Left two comments in the Figma file. Please have a look.

**Comment by beatrizmartinmartins** [comment]
@inesdgomes responded on the comments. Please have a look so I can make the changes

**Comment by inesdgomes** [comment]
@beatrizmartinmartins Done. Please let me know if there's anything unclear.

**Comment by beatrizmartinmartins** [comment]
@inesdgomes I also saw a few comments on the templates file and you are right, it looks way too big on the Publications when we include 64px after each section. Should we include just one 64px spacing? The thing Im thinking is, how will we make this as an exception? Can we make them only for Publications pages since on the other pages it works just fine?

**Comment by inesdgomes** [comment]
@beatrizmartinmartins Yes, we can make exceptions per template. I think that's more related to this ticket: https://github.com/orgs/international-labour-organization/projects/2/views/14?pane=issue&itemId=67152153

Regarding this task, I think we still need to address this comment: https://www.figma.com/design/qr8jQnxqJkRJFQnVC5PmrB?node-id=1905-4#921556297

**Comment by beatrizmartinmartins** [comment]
@inesdgomes changes the List component to add a short list and a long list option like we talked about. See [here](https://www.figma.com/design/RRxsfLNxjoypKrfJg1EwJf/ILO-Components-Library?m=auto&node-id=1590-13006&t=fL5D3MvkmHd4f0Pp-1)

**Comment by inesdgomes** [comment]
@beatrizmartinmartins I think we're almost there. There's still this comment to resolve: https://www.figma.com/design/qr8jQnxqJkRJFQnVC5PmrB?node-id=1905-4#936751119

**Comment by beatrizmartinmartins** [comment]
I think this is done on my side @inesdgomes

**Comment by inesdgomes** [comment]
Indeed @beatrizmartinmartins I think it's all done from the design side.
@justintemps I think you're working on this as part of another ticket, but assigning it to you to ensure the updated documentation on spacing is reflected on the rich text component.

**Comment by justintemps** [comment]
Hi @inesdgomes this should have been done as part of #1233. If that's the case, then the revised spacings will already be in production on ilo.org. Could you check and close this ticket if it's the case?

**Comment by inesdgomes** [comment]
Yes, this is done. Closing this ticket.
